### PR TITLE
Coding Standards from Ticket 49542

### PR DESCRIPTION
**Pull request by mattyrob**
## Description
This PR includes all relevant changes included in upstream ticket 49542:
https://core.trac.wordpress.org/ticket/49542
Not all of the 37 commits were necessary or needed, some applied to files not included in ClassicPress

## Motivation and context
Improve coding standards for readability and to aid move to PHP 8 and 8.1 compatibility.
It also commences work as described in the discussion on #1098 

## How has this been tested?
Local PHPUnit and `phpcs` is passing and a quick manual localhost test looks good.

## Screenshots
N/A

## Types of changes
- Bug fix / Enhancement

### CODE REVIEW

File: src/wp-admin/includes/class-wp-filesystem-ftpext.php [hunk 1]
```diff
@@ -122,28 +126,29 @@ public function connect() {
...
 		$contents = '';
 
-		while ( ! feof( $temp ) ) {
-			$contents .= fread( $temp, 8192 );
+		while ( ! feof( $temphandle ) ) {
+			$contents .= fread( $temphandle, 8 * KB_IN_BYTES );
 		}
 
-		fclose( $temp );
+		fclose( $temphangle );
```
Comment (xxsimoxx) [inline src/wp-admin/includes/class-wp-filesystem-ftpext.php:149]: Here there is a typo in the var.

**Review by xxsimoxx — CHANGES_REQUESTED** [review]

File: src/wp-admin/includes/class-wp-filesystem-ftpext.php (hunk 1 above)
Comment (mattyrob) [reply src/wp-admin/includes/class-wp-filesystem-ftpext.php]: Thanks, good catch, updated PR to follow. There probably is a sniff for unused variables available, would have to investigate.

**Review by xxsimoxx — APPROVED** [review]
Tested on a live site for some days.
LGTM.
Also fix issue that causes problems with front static page in PHP 8 that was previously addressed by #1067.

**Comment by mattyrob** [comment]
Reviewed at PR meeting and agreed for merge

**Pull request merged**